nodes/staking/Cargo.toml

```diff
@@ -21,7 +21,7 @@ path = "src/main.rs"
 # third-party dependencies
 jsonrpc-core = "18.0.0"
 clap = { version = "3.0", features = ["derive"], optional = true }
-codec = { package = "parity-scale-codec", version = "2.0.0" }
+codec = { package = "parity-scale-codec", version = "3.0.0", default-features = false }
```

Comment:
`default-features = false` is not needed.

Reply:
corrected it